src/abi/fuse_abi.rs

```diff
@@ -612,9 +612,17 @@ impl Attr {
             mode: st.st_mode as u32,
             // st.st_nlink is u64 on x86_64 and powerpc64, and u32 on other architectures
             // ref https://github.com/rust-lang/rust/blob/1.69.0/library/std/src/os/linux/raw.rs#L333
-            #[cfg(any(target_arch = "x86_64", target_arch = "powerpc64"))]
+            #[cfg(any(
+                target_arch = "x86_64",
+                target_arch = "powerpc64",
+                target_arch = "aarch64"
```

Comment:
This will break Linux aarch64 where `st.st_nlink` is `u64`.

Also the commit title says that `st.st_nlink` is u16 macos aarch64, but the code ensures that it is `u32`?

Reply:
Only `as u32` if target_os macos.

Reply:
And `as` mean cast not ensure type?